redirects-transition.conf

```diff
@@ -480,23 +480,26 @@ rewrite ^/law/policy/embezzle/embezzlementpolicycomments.shtml https://www.fec.g
 # law/policy/enforcement/ redirects
 rewrite ^/law/policy/enforcement/publichearing011409.shtml https://www.fec.gov/updates/january-14-15-2009-public-hearing-agency-procedures/ redirect;
 rewrite ^/law/policy/enforcement/2013/2013commentsreceived.shtml https://www.fec.gov/legal-resources/policy/comments-received-enforcement-process-2013/ redirect;
+rewrite ^/law/policy/enforcement/hearing/notice2013-01.pdf https://www.fec.gov/resources/cms-content/documents/notice2013-01.pdf redirect;
```

Comment:
One small fix to put things in alphabetical order: Move the redirect at line 483 (rewrite ^/law/policy/enforcement/hearing/notice2013-01.pdf https://www.fec.gov/resources/cms-content/documents/notice2013-01.pdf redirect;) to the top of the law/policy/enforcement category - it should come first alphabetically.